app/views/users/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div id="review-list">
+  <% @user.reviews.each do |review| %>
+  <div id="book-<%= review.book.id %>-info">
+    <img src="<%= review.book.thumbnail %>" alt="Cover of <%= review.book.title %>">
+    <br><h3><%= review.book.title %></h3><br>
+    <!-- possible format change for readability -->
+  </div>
+  <div id="review-<%= review.id %>-info">
+    <h3><%= review.title %></h3>
+    <p>Written by: <%= @user.username %></p>
```

Comment:
```suggestion

```

Reply:
Since we're already on a particular user's show page, I don't think we need to include this for each review.